src/design-tokens/tier-1-definitions/typography.json

```diff
@@ -3,13 +3,13 @@
     "typography": {
       "fontFamily": {
         "1": {
-          "value": "'Parabolica', Arial, sans-serif"
+          "value": "'parabolica', Arial, sans-serif"
```

Comment:
Case and spelling must match the naming in the linked CSS files containing the `@font-face` declarations.